autoload/codefmt.vim

```diff
@@ -252,6 +262,23 @@ function! codefmt#GetSupportedFormatters(ArgLead, CmdLine, CursorPos) abort
   return join(l:groups[0] + l:groups[1] + l:groups[2], "\n")
 endfunction
 
+""
+" Returns whether there is a default formatter available for the current
+" buffer.
+function! codefmt#AvailableInCurrrentBuffer() abort
+  let l:formatters = s:registry.GetExtensions()
+  if !empty(get(b:, 'codefmt_formatter'))
+    let l:Predicate = {f -> f.name ==# b:codefmt_formatter}
+  else
+    let l:Predicate = {f -> f.AppliesToBuffer() && s:IsAvailable(f)}
+  endif
```

Comment:
For reference: We haven't been using lambdas in any of our plugins yet, but I checked and they were originally added in 7.4.2044 from 2016. That's probably long enough ago. At some point I should probably declare a min supported vim version for the plugin and get it testing against that version, though.